packages/vuetify/src/components/VFileInput/__tests__/VFileInput.spec.ts

```diff
@@ -231,4 +231,34 @@ describe('VFileInput.ts', () => {
 
     expect(change).not.toHaveBeenCalled()
   })
+
+  it('should truncate correctly', () => {
+    const fifteenCharFile = { name: 'testFile15Chars', size: 1 }
+    const wrapper = mountFunction({
+      propsData: {
+        truncateLength: 1,
+        value: fifteenCharFile,
+      },
+    })
+
+    expect(wrapper.find('.v-file-input__text').text()).toBe('…')
+
+    wrapper.setProps({
+      truncateLength: 2,
```

Comment:
What's about leaving char in the beginning here? I'm not sure if it's a good idea

Reply:
Yeah I also wanted to discuss this. Would you say it's better if we completely hide the filename if user specifies `truncateLength` to be 1?

Reply:
I don't know